data/templates/esx-ks

```diff
@@ -7,28 +7,6 @@ accepteula
   install --disk=<%=installDisk%> --overwritevmfs
 <% } %>
 rootpw <%=rootPlainPassword%>
-
-# Search the networkDevices and set the first device (if defined) up.
-# If no device is specified in the networkDevices, then we fallback
-# to setting 'vmnic0' up as DHCP. The device can be specified with a
-# MAC address or device name ('vmnic0' for example)
-<% if( typeof networkDevices !== 'undefined' ) { %>
-  <% need_default = networkDevices.every(function(n) { %>
-    <% if(typeof n.ipv4 !== 'undefined') { %>
-      <% ipopts = '--ip=' + n.ipv4.ipAddr + ' --gateway=' + n.ipv4.gateway + ' --netmask=' + n.ipv4.netmask %>
-      <% if (typeof n.ipv4.vlanIds !== 'undefined' ) { %>
-        <% ipopts += ' --vlandid=' + n.ipv4.vlanIds[0] %>
-      <% } %>
-      network --bootproto=static --device=<%=n.device%> <%=ipopts%>
-    <% } else { %>
-      network --bootproto=dhcp --device=<%=n.device%>
-    <% } %>
-    <% return false; %>
-  <% }); %>
-  <% if (need_default) { %>
-    network --bootproto=dhcp --device=vmnic0
-  <% } %>
-<% } %>
```

Comment:
wouldn't the first boot section take care of that? Even if dhcp is set for installation, once firstboot runs the ip will be set to static as desired.